### PR TITLE
compact error report

### PR DESCRIPTION
**Pull request by eishay**

### CODE REVIEW

File: shoebox/app/com/keepit/common/healthcheck/Healthcheck.scala [hunk 1]
```diff
@@ -27,9 +29,19 @@ import scala.concurrent.duration._
 import com.keepit.common.akka.FortyTwoActor
 import com.keepit.common.plugin.SchedulingPlugin
 
+case class HealthcheckErrorSignature(value: String) extends AnyVal
+
 case class HealthcheckError(error: Option[Throwable] = None, method: Option[String] = None,
     path: Option[String] = None, callType: CallType, errorMessage: Option[String] = None,
     id: ExternalId[HealthcheckError] = ExternalId(), createdAt: DateTime = currentDateTime) {
+
+  lazy val formattedStackTrace = error.map(e => e.getStackTrace() mkString "\n<br/> &nbsp; ").getOrElse("")
```
Comment (2is10) [inline shoebox/app/com/keepit/common/healthcheck/Healthcheck.scala]: maybe `stackTraceHtml`? (more specific)

**Comment by eishay** [comment]
formattedStackTrace -> stackTraceHtml

File: shoebox/conf/routes [hunk 2]
```diff
@@ -167,6 +167,7 @@ POST    /admin/phrases/save         com.keepit.controllers.admin.PhraseControlle
 
 GET     /admin/healthcheck/ping     com.keepit.common.healthcheck.HealthController.ping()
 GET     /admin/healthcheck/error    com.keepit.common.healthcheck.HealthController.causeError()
+GET     /admin/healthcheck/reportErrors    com.keepit.common.healthcheck.HealthController.reportErrors()
 GET     /admin/healthcheck/errors   com.keepit.common.healthcheck.HealthController.getErrors()
 GET     /admin/healthcheck/isHealthy    com.keepit.common.healthcheck.HealthController.isHealthy()
 GET     /admin/healthcheck/resetErrorCount    com.keepit.common.healthcheck.HealthController.resetErrorCount()
```
Comment (2is10) [inline shoebox/conf/routes:173]: Might be worth adding a brief comment explaining why the requests that have side effects here use GET instead of POST.

**Comment by 2is10** [comment]
This will be very helpful. Thanks.

**Pull request merged**